contrib/cf/manifest.yml

```diff
@@ -15,12 +15,14 @@ applications:
     STORAGE_REDIS_PORT: 6380
     STORAGE_REDIS_DB: 0
     STORAGE_REDIS_ENABLE_TLS: true
+    STORAGE_REDIS_PREFIX:
     CRYPTO_ENCRYPTION_SCHEME: AES256
     CRYPTO_AES256_KEY: AES256Key-32Characters1234567890
     ASYNC_REDIS_HOST: <HOSTNAME FROM AZURE REDIS CACHE>
     ASYNC_REDIS_PASSWORD: <PRIMARYKEY FROM AZURE REDIS CACHE>
     ASYNC_REDIS_PORT: 6380
     ASYNC_REDIS_DB: 1
+    ASYNC_REDIS_PREFIX:
```

Comment:
Looks like in our Helm chart, we only make `REDIS_PREFIX` settable. (In the code, both storage and async will fall back on `REDIS_PREFIX` if they can't find `STORAGE_REDIS_PREFIX` and `ASYNC_REDIS_PREFIX`, respectively.)

I think we should _also_ only expose `REDIS_PREFIX` here. The reason is that this prevents those two settings from being tuned independently and forces both to have the same value-- which imo is enormously useful for helping to maintain a correlation / traceability by grouping all keys for a given OSBA deployment together.

Reply:
Yeah, great point. I'll make that change,.